wmn-data.json

```diff
@@ -3473,6 +3473,16 @@
         "known" : ["test", "janek"],
         "cat" : "images"
        },
+        {
+        "name" : "Moto Trip",
+        "uri_check" : "https://moto-trip.com/profil/{account}",
+        "e_code" : 200,
+        "e_string" : "Profil de:",
+        "m_string" : "Page introuvable La page que vous demandez n'existe pas ou plus.",

```

Comment:
I also don't get this string for a bad user. Check the page source code. Maybe change to `<h1>Page introuvable</h1>`